version.sbt

```diff
@@ -0,0 +1 @@
+ThisBuild / version := "0.0.1-SNAPSHOT"
```

Comment:
sbt-typelevel has a git-based versioning scheme. You should add `ThisBuild / tlBaseVersion := "0.0"` to the primary build.
```suggestion
```